tests/integration/test_backup_restore_new/test.py

```diff
@@ -1206,10 +1206,27 @@ def test_backup_all(exclude_system_log_tables):
 
     exclude_from_backup = []
     if exclude_system_log_tables:
-        system_log_tables = instance.query(
-            "SELECT concat('system.', table) FROM system.tables WHERE (database = 'system') AND (table LIKE '%_log')"
```

Comment:
This query didn't work well because these logs tables are created in background so some log tables could be created after this query but before the `BACKUP ALL EXCEPT ...` query captured a list of tables to backup. And that caused the test to fail. So I decided to just hardcode the whole list of system log tables.